client/python/lib/test/config.py

```diff
@@ -30,3 +30,232 @@
+    ({"client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, ValueError, "none or both client_key_path and client_cert_path are required in tls_config", 0),
+    
+    ({"non_client_key_path" : PATH_VALID, "client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, ValueError, "non_client_key_path is not valid tls_config key", 0),
+    ({"client_key_path" : PATH_VALID, "non_client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, ValueError, "non_client_cert_path is not valid tls_config key", 0),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : PATH_VALID, "non_server_cert_path" : PATH_VALID}, ValueError, "server_cert_path is not defined in tls_config", 0),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : PATH_VALID}, ValueError, "server_cert_path is not defined in tls_config", 0),
+
+    ({"client_key_path" : 1, "client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, TypeError, f'client_key_path type should be string but is type int', 0),
+    ({"client_key_path" : [PATH_VALID], "client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, TypeError, f'client_key_path type should be string but is type list', 0),
+    ({"client_key_path" : 1, "client_cert_path" : PATH_VALID, "server_cert_path" : 1}, TypeError, f'client_key_path type should be string but is type int', 0),
+
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : 1, "server_cert_path" : PATH_VALID}, TypeError, f'client_cert_path type should be string but is type int', 1),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : 1, "server_cert_path" : 1}, TypeError, f'client_cert_path type should be string but is type int', 1),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : [PATH_VALID], "server_cert_path" : PATH_VALID}, TypeError, f'client_cert_path type should be string but is type list', 1),
+
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : PATH_VALID, "server_cert_path" : 1}, TypeError, f'server_cert_path type should be string but is type int', 2),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : 1, "server_cert_path" : [PATH_VALID]}, TypeError, f'client_cert_path type should be string but is type int', 1),
+    
+    ({"client_key_path" : "invalid", "client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, ValueError, f'invalid is not valid path to file', 1),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : "/very/invalid", "server_cert_path" : PATH_VALID}, ValueError, f'/very/invalid is not valid path to file', 2),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : PATH_VALID, "server_cert_path" : "third_invalid_path"}, ValueError, f'third_invalid_path is not valid path to file', 3),
+
+]
+
+CONFIG_VALID = [
+    ({
+        "address" : "localhost",
+        "port" : 9000
+    }, {"check_address" : 1, "check_port" : 1, "check_tls_config" : 0}),
```

Comment:
I assume those 1 and 0 means valid/invalid here. Can we have it named in the const and applied here and in all cases?